.circleci/config.yml

```diff
@@ -555,7 +555,17 @@ jobs:
           command: |
             echo "SUCCESS"
 
-# docker push
+  pull-coredns:
+    docker:
+      - image: circleci/golang
```

Comment:
NIT Interesting, do we need to use the golang image here, just out of curiosity. I guess it just works.

Reply:
done

Reply:
I suppose for generic stuff we could use alpine, but I suppose any circleci/image should work fine :-)